assignments/callbacks.js

```diff
@@ -1,6 +1,8 @@
 // Create a higher order function and invoke the callback function to test your work. You have been provided an example of a problem and a solution to see how this works with our items array.  Study both the problem and the solution to figure out the rest of the problems.
 
-const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
+const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum', 'Icecream', 'donut'];
+const countries = ['Nigeria', 'Ghana', 'Togo', 'Benin', 'Ivory Coast']
+let list = ['Nigeria', 'Ghana', 'Togo', 'Benin', 'Ivory Coast']
```

Comment:
Here you can give our variables more descriptive names to let another developer looking at your code understand what the variables contain. You could call the `list` variable `westAfricanCountries` for example; a long and descriptive variable name is a better variable name. 👍